server/methods/catalog.js

```diff
@@ -398,10 +399,10 @@ Meteor.methods({
             `${variant.compareAtPrice}`
         });
       } else {
-        const parentIndex = sortedVariant.ancestors.indexOf(variantId);
+        const parentIndex = sortedVariant.ancestors.includes(variantId);
```

Comment:
Since the actual `parentIndex` value is used for the splice below, this needs to stay `indexOf`. You can change the line below to

```js
if (parentIndex >= 0) ancestorsClone.splice(parentIndex, 1, variantNewId);
```

Reply:
@aldeed I believe this [commit](https://github.com/reactioncommerce/reaction/pull/3599/commits/d7c26811445f9c6f6e40d21be19fd4c179d60958) is fixing the issue.